resource/reactivedrop_swedish.txt

```diff
@@ -1029,7 +1029,7 @@
 "[english]asw_take_ammo_vindicator"		"Take Vindicator Ammo"
 "asw_take_ammo_vindicator"		"Ta Vindicator-ammo"
 "[english]asw_take_ammo_flamer"		"Take Flamer Ammo"
-"asw_take_ammo_flamer"		"Ta Låg-ammo"
+"asw_take_ammo_flamer"		"Ta eldkastar-ammo"
```

Comment:
The flamethrower is named "lågantändning" in this game, thats why it says låg-ammo.